packages/feedback-react/src/followupContext.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { createContext, useContext } from "react";
+import { FollowupState } from "./types";
+
+const initialValue: FollowupState = {
+    handleNext: () => null,
+    handleAbort: () => null,
+    step: {
+        number: 0,
+        isLast: false,
+        question: {
+            label: "",
+            options: [],
+            type: "text",
+        },
+    },
+    submitted: false,
+    values: [],
+};
+
+const followUpContext = createContext<FollowupState>(initialValue);
+
+export const useFollowUpContext = () => useContext(followUpContext);
+
+export const FollowUpProvider: React.FC<{ state: FollowupState }> = ({ state, children }) => (
```

Comment:
Litt pirk, men det er litt miks av stor og liten u.
Det er forsåvidt greit, for ved verb er det to ord (to follow up), mens som substantiv er det med bindestrek (a follow-up).
Jeg kommenterte derfor ikke på `followUp` som prop til `<Feedback />`, den er jeg enig at skal ha stor U.
Her derimot burde alt ha liten u, slik som også filnavnet har. `followUpContext`, `useFollowUpContext` og `FollowUpProvider` mener jeg altså at alt burde ha liten u.
Moro med språkpirk :laughing: :nerd_face:

Reply:
Beklager, jeg tenkte i sta at det var en boolean, altså som i verbform "should we follow up?".
Jeg så nå at `followUp` er et objekt som beskriver en follow-up. Altså burde det der også være liten u.

Reply:
Ja det er litt miks 😑 Jeg ville imidlertid påstått at ord med bindestrek mellom er å regne som to ord, og at det derfor er riktig med stor "U", men jeg ser nå at konvensjonen er å ikke bruke stor bokstav! `followup` it is, then!

Reply:
> Jeg ville imidlertid påstått at ord med bindestrek mellom er å regne som to ord

Litt kjapp googling peker mot at ord med bindestrek regnes som ett ord.
Dette er altså ikke kunnskap jeg sitter inne med, men som jeg synes er litt moro å finne ut av :nerd_face: 
Bra vi er enige om `followup`, da trenger vi ingen follow-up i den saken :smile: